odoo/import_xml.rng

```diff
@@ -214,6 +214,24 @@
         </rng:element>
     </rng:define>
 
+    <rng:define name="asset">
+        <rng:element name="asset">
+            <rng:attribute name="id" />
+            <rng:attribute name="name" />
+            <rng:optional><rng:attribute name="active" /></rng:optional>
+            <rng:optional><rng:attribute name="website_id" /></rng:optional>
```

Comment:
Do we need this one? It's probably something we won't be using, like for views, I am surprised it exists, probably some old stuff not used anymore
I'd leave that to the normal `<field/>` behavior, no need to add it as attribute capability IMO